hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -342,6 +342,8 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=80_000_000", "+calibrate_usb_clk=1"]
       reseed: 20
+      // slow baudrate may be used, it needs longer time
+      run_timeout_mins: 120
```

Comment:
@matutem this doesn't work due to the error below.
how can I override it for a test?

> ERROR: [Modes] Mode sw_test_mode_test_rom cannot be merged into chip_sw_uart_rand_baudrate due to a conflict (unable to pick one from different values): run_timeout_mins: {120, 60}

Reply:
Filed an issue #14092

Reply:
I will suggest running this on our cloud infra as well, to ensure the new timeout is indeed getting adhered to.

Reply:
Sure, it looks ok, but let me wait until the test is finished.